istio/src/main/protobuf/http_fault.proto

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2017 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+syntax = "proto3";
+
+import "duration.proto";
```

Comment:
I bet we could do that, makes sense if we need to import a bunch more. However I also edited the google protobuf files to change their package `option java_package = "com.google.local";` otherwise they conflict with java's protoc lib. If you figure out a better solution, I'm all ears.